docs/developer/services/hello-world/helloworld-ruby/index.md

```diff
@@ -245,4 +247,4 @@ To remove the sample app from your cluster, delete the service record.
   ```
 
 {{< /tab >}}
-{{< /tabs >}}
\ No newline at end of file
+{{< /tabs >}}
```

Comment:
Do you know what this shortcode does?? For website update purposes

Reply:
Adds tabs 🙃